platform/iphone/view_controller.mm

```diff
@@ -129,6 +129,14 @@ - (BOOL)prefersStatusBarHidden {
 	return YES;
 }
 
+- (BOOL)prefersHomeIndicatorAutoHidden {
+	if(OS::get_singleton()->get_hide_home_indicator()) {
```

Comment:
Style issue, there should be a space after `if`.

Reply:
Fixed